Lib/fontTools/misc/timeTools.py

```diff
@@ -47,7 +48,12 @@ def timestampFromString(value):
 	return calendar.timegm(time.strptime(value)) - epoch_diff
 
 def timestampNow():
-	return int(time.time() - epoch_diff)
+	# https://reproducible-builds.org/specs/source-date-epoch/
+	source_date_epoch = os.environ.get("SOURCE_DATE_EPOCH")
+	if source_date_epoch is not None:
+		return int(int(source_date_epoch) - epoch_diff)
```

Comment:
i believe `epoch_diff` is already an int, because that's what `calendar.timegm()` returns, so I think you don't need to call `int` twice.

Reply:
The existing code did an `int` cast as well, since I couldn’t tell why I chose the safe said and did the same.

Reply:
the reason the original code was doing that is because `time.time()` returns a float, and subtracting a int from a float yields a float.

Reply:
I just found this out 😄